configtx/example_test.go

```diff
@@ -132,21 +132,21 @@ func Example_basic() {
 }
 
 // This example updates an existing orderer configuration.
-func ExampleConfigTx_SetOrdererConfiguration() {
+func ExampleUpdatedOrdererGroup_SetConfiguration() {
 	baseConfig := fetchChannelConfig()
 	c := configtx.New(baseConfig)
 
 	// Must retrieve the current orderer configuration from block and modify
 	// the desired values
-	orderer, err := c.OrdererConfiguration()
+	orderer, err := c.UpdatedConfig().Orderer().Configuration()
 	if err != nil {
 		panic(err)
 	}
 
 	orderer.Kafka.Brokers = []string{"kafka0:9092", "kafka1:9092", "kafka2:9092"}
 	orderer.BatchSize.MaxMessageCount = 500
 
-	err = c.SetOrdererConfiguration(orderer)
+	err = c.UpdatedConfig().Orderer().SetConfiguration(orderer)
```

Comment:
The repeated use of the chain does not demonstrate how I'd expect this to be used IRL. Why isn't the orderer reference obtained and maintained across the calls?